### PR TITLE
Transfer: Move conveyor transfers to third_party_copy_operation: Close #535

### LINKED ISSUE #535: Move conveyor transfers to third_party_copy_operation
Motivation
----------
At the moment read is used for source and write is used for destination, for both third_party_copy should be used.

### PR DESCRIPTION
**Pull request by cserf**
Transfer: Move conveyor transfers to third_party_copy_operation: Close #535

### CODE REVIEW

File: lib/rucio/core/rse.py [hunk 1]
```diff
@@ -757,7 +753,6 @@ def get_rse_protocols(rse, schemes=None, session=None):
             'rse_type': str(_rse.rse_type),
             'credentials': None,
             'volatile': _rse.volatile,
-            'verify_checksum': verify_checksum[0] if verify_checksum else True,
```
Comment (mlassnig) [inline lib/rucio/core/rse.py]: hmmmm, did you remove this on purpose? i thought frank put this explicitly to make webdav transfers work.

**Review by mlassnig — COMMENTED** [review]
this matches src:read_wan with dst:third_party_copy, correct?

File: lib/rucio/core/transfer.py [hunk 2]
```diff
@@ -559,16 +559,15 @@ def get_transfer_requests_and_source_replicas(process=None, total_processes=None
                 allow_tape_source = True
 
                 # Find matching scheme between destination and source
-                # TODO: In the future, move to domain "third_party_copy"
                 try:
                     matching_scheme = rsemgr.find_matching_scheme(rse_settings_dest=rses_info[dest_rse_id],
                                                                   rse_settings_src=rses_info[source_rse_id],
                                                                   operation_src='read',
```
Comment (bari12) [inline lib/rucio/core/transfer.py]: I was under the impression we do third_part_copy to third_party_copy, thus also using this as the source operation?

File: lib/rucio/core/rse.py (hunk 1 above)
Comment (cserf) [reply lib/rucio/core/rse.py]: Oups, no. Will put it back

File: lib/rucio/core/transfer.py (hunk 2 above)
Comment (mlassnig) [reply lib/rucio/core/transfer.py]: we should, that's why i asked ;-)

File: lib/rucio/core/rse.py (hunk 1 above)
Comment (mlassnig) [reply lib/rucio/core/rse.py]: also, line 742 above, just saw it

File: lib/rucio/core/transfer.py (hunk 2 above)
Comment (mlassnig) [inline lib/rucio/core/transfer.py]: this should also be third_party_copy

**Review by mlassnig — DISMISSED** [review]

**Comment by bari12** [comment]
@cserf please rebase

**Review by bari12 — APPROVED** [review]

**Pull request merged**